test/TestStore.js

```diff
@@ -0,0 +1,27 @@
+'use strict'
+
+const { EventEmitter } = require('events')
+
+class TestStore extends EventEmitter {
```

Comment:
why do we need this? Looks identical to https://github.com/fastify/session/blob/master/lib/store.js except being a `class`

Reply:
simplified it. it differs from the .set()-method.

Reply:
Also I dont want that somebodys code breaks just because I modified the Store for the tests. Imagine somebody really using the memorystore for some reason and somehow stores circular references or so. Also it makes it possible to later modify the store to be a RedisStore or so.

Reply:
nice, thanks for clarifying!